presto-main/src/main/java/com/facebook/presto/sql/relational/FunctionResolution.java

```diff
@@ -260,7 +260,7 @@ public FunctionHandle tryFunction(Type returnType)
 
     public boolean isTryFunction(FunctionHandle functionHandle)
     {
-        return functionAndTypeResolver.getFunctionMetadata(functionHandle).getName().equals("$internal$try");
+        return functionAndTypeResolver.getFunctionMetadata(functionHandle).getName().getObjectName().equals("$internal$try");
```

Comment:
Which changes required this change? Or it was just a bug?

Reply:
It was a bug, comparing QualifiedObjectName and String...